_automating-workflows/api/get-workflow-steps.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Get workflow steps
+parent: Workflow API
+nav_order: 50
+---
+
+# Get workflow steps
+
+OpenSearch validates workflows using the [JSON file](https://github.com/opensearch-project/automating-workflows/blob/main/src/main/resources/mappings/workflow-steps.json) that lists the required inputs and generated outputs for all steps. The Get Workflow Steps API retrieves this file.   
```

Comment:
Links to the github repo should have `flow-framework` rather than `automating-workflows`.

But I'm not sure we should refer to this file by link, particularly on the `main` branch:
 - We have an open issue to store it programmatically and eliminate the file, which almost certainly will be done before 2.13.
 - Any new steps we add will change this file on `main` for upcoming versions but wouldn't be reflected in the released API which includes this file as a classpath resource.